### PR TITLE
shashvat

### PR DESCRIPTION
**Pull request by Shashvat-Jain**

### CODE REVIEW

**Review by Ayush1325 — CHANGES_REQUESTED** [review]
Please make it obvious which function you expect me to call. Name it `main` or something.

Also add type hints for me to perform appropriate conversions.

**Review by Shashvat-Jain — COMMENTED** [review]
Please review

File: calculator_gui/tasks/lagrange_coeff.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+def ss(z):
+    if z > 0:
+        return (math.sqrt(z) - math.sin(math.sqrt(z))) / (math.sqrt(z)) ** 3
+    if z < 0:
+        return (math.sinh(math.sqrt(-z)) - math.sqrt(-z)) / (math.sqrt(-z)) ** 3
+    else:
+        return 1 / 6
+
+
+def main(m, x, t, r, ro, a):
```
Comment (Ayush1325) [inline calculator_gui/tasks/lagrange_coeff.py]: What are the data types of `m, x, t, r, ro, a`. Add [type hints](https://docs.python.org/3/library/typing.html)

File: calculator_gui/tasks/lambert.py [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+import numpy as np
+from numpy import cross, pi
+from numpy.linalg import norm
+from scipy.special import hyp2f1
+
+# from lamberthub.utils.assertions import assert_parameters_are_valid
+
+
+def main(
```
Comment (Ayush1325) [inline calculator_gui/tasks/lambert.py:17]: Same as above

**Review by Ayush1325 — CHANGES_REQUESTED** [review]
Fix Pylint CI errors. You can run pylint locally as well:
```shell
pylint $(git ls-files '*.py')
```

**Comment by Ayush1325** [comment]
Manual merge

**Pull request closed without merging**